tests/python/unittest/test_optimizer.py

```diff
@@ -512,7 +512,10 @@ def update(self, index, weight, grad, state):
         prev_v[:] = v_t
         prev_z[:] = z_t
 
-@with_seed(0)
+# @mseth10: Getting rid of fixed seed. Flakiness was reproduced on GPU.
+# rtol and atol increased to 10x their default values to resolve it.
+# can be tracked at https://github.com/apache/incubator-mxnet/issues/11735
```

Comment:
Feel free to remove those 3 lines of comment and simply link the issue in the description of the PR.